layout-normalizer/src/gpos.rs

```diff
@@ -228,6 +239,22 @@ impl ResolvedValue {
     fn is_zero(&self) -> bool {
         self.default == 0 && self.device_or_deltas.is_none()
     }
+
+    fn add_in_place(&mut self, other: &ResolvedValue) {
+        self.default += other.default;
+        //CODE REVIEW:
+        // - if we have device tables we're just dropping the second, which
+        // is fine for our purposes? And if we have mixed deltas & devices than
+        // god help us
```

Comment:
this is fine, I've actually never seen device tables in the wild. how about we at least warn?

Reply:
we can warn, but when we run with `ttx_diff` (which is the most common case) we suppress stderr so we might never see it. 🤔

Reply:
This is an internal tool, and we don't use device tables, so I'm just going to let this slide.